src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQueryOutputFormat.java

```diff
@@ -187,6 +186,10 @@ public void commitJob(JobContext jobContext) throws IOException {
       LOG.debug("Allow schema relaxation: '{}'", allowSchemaRelaxation);
       PartitionType partitionType = conf.getEnum(BigQueryConstants.CONFIG_PARTITION_TYPE, PartitionType.NONE);
       LOG.debug("Create Partitioned Table type: '{}'", partitionType);
+      com.google.cloud.bigquery.TimePartitioning.Type timePartitioningType = conf.getEnum(
```

Comment:
can't we import this `Type`.

Reply:
> It's better that you comment on which one to choose.

Here's my reason.

`TimePartitioning` is the class and `Type` is an enum within it.

- Can't import `TimePartitioning` as other class with the same name is being imported
- Importing Type is a possibility but as everywhere else i have used `TimePartitioning.Type` this will make using `Type` not consistent 
- Using `Type` as the variable type feels weird

Reply:
Use TimePartitioning.Type